layouts/partials/meta.html

```diff
@@ -1,10 +1,14 @@
-<title>{{if .IsHome}}{{ .Title }}{{else}}{{ .Title }} - {{ .Site.Title }}{{end}}
+<title>
+  {{ if .IsHome }}
+      {{ .Site.Title }} - Food Spraying Technology
+  {{ else }}
+      {{ .Title }} - {{ .Site.Title }}
+  {{ end }}
```

Comment:
Put this back the way it was. You should not hardcode values.
This is why the title is currently showing wrong on the home page